src/lib/components/datatable/Row.svelte

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+  import type { Column } from '$lib/models/Tables';
+  import type { Indexable } from '$lib/types';
+  import Search from '$lib/stores/Search';
+  let { activeRow, expandableComponents, activeComponent } = Search;
```

Comment:
Might want to move these active things outside the search - the table isn't just used in the search context. Maybe a TableControlStore or something.